src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
 public class WorkWithFile {
+    private static final String ADDED = "buy";
+    private static final String LOSE = "supply";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        HashMap<String, Integer> sellAndBuy = new HashMap<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String lineWords = bufferedReader.readLine();
+            while (lineWords != null) {
+                String[] line = lineWords.split(",");
```

Comment:
```suggestion
                String[] line = lineWords.split(CSV_SEPARATOR);
```
let's declare it as constant